app/models/rook.rb

```diff
@@ -1,2 +1,7 @@
 class Rook < Piece
+  def to_ascii_piece
```

Comment:
To handle color - check the color on the piece model and return the code accordingly

```
if self.color == 'black'
  # return black code
else
  # return white code
end
```